iati/about/models.py

```diff
@@ -64,18 +66,15 @@ def get_context(self, request, *args, **kwargs):
     ]
 
 
-class CaseStudyPage(AbstractContentPage):
+class CaseStudyPage(AbstractContentPage):  # pylint: disable=too-many-ancestors
     """A model for Case Study pages."""
 
     parent_page_types = ['about.CaseStudyIndexPage']
     subpage_types = []
 
     feed_image = models.ForeignKey(
-        'wagtailimages.Image',
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
-        related_name='+',
+        'wagtailimages.Image', null=True, blank=True,
```

Comment:
As the previous state for this looked cleaner and was slightly more readable, we'd prefer to mark this for the linters to ignore over the changes made here.

Reply:
This applies to the same changes elsewhere too.

Reply:
> we'd prefer to mark this for the linters to ignore

^^ I should have explained – I think it’s only possible to do this globally. I don’t think it’s possible to ignore `duplicate-code` errors case by case.

Reply:
For reference: https://github.com/PyCQA/pylint/issues/214

Gah! That's annoying. Will accept this change then as there don't seem to be any better options.

Reply:
Thanks for adding the ref! Kk I’ll leave this one.